tests/filecheck/transforms/copy_to_dma.mlir

```diff
@@ -0,0 +1,23 @@
+// RUN: ./compiler/snax-opt %s -p snax-copy-to-dma --print-op-generic | filecheck %s
+
+"builtin.module"() ({
+  "func.func"() <{"sym_name" = "simple_mult", "function_type" = (memref<?xi32>, memref<?xi32>) -> (), "sym_visibility" = "public"}> ({
+  ^0(%arg0 : memref<?xi32>, %arg1 : memref<?xi32>):
+    "memref.copy"(%arg0, %arg1) : (memref<?xi32>, memref<?xi32>) -> ()
+    "func.return"() : () -> ()
+  }) : () -> ()
+}) : () -> ()
```

Comment:
Can we also add a static test case and check whether mlir-opt can bypass the extract_aligned_pointer etc. immediately? I have the feeling these should be constant folded

Reply:
Yes, but this happens during inlining or at the LLVM level, so not really a test for this transform?